Src/Common/Domain/service/IOrganizationUserService.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Ttu.Domain
+{
+    public interface IOrganizationUserService
```

Comment:
what's the "userId" parameter in these? Should that be named userRecordId? If the userId you are passing in is the logged in user record id, then you can just get that in the Service class with UnitOfWork.User